ci-src-requirements.txt

```diff
@@ -1 +0,0 @@
-git+http://github.com/enthought/traitsui.git#egg=traitsui
```

Comment:
Then I believe you will need to add this back to `source_dependencies` in `etstool.py` for the cron job to run against master.